plugins/cue_points/quiz/lib/api/filters/KalturaQuizUserEntryFilter.php

```diff
@@ -5,4 +5,28 @@
  */
 class KalturaQuizUserEntryFilter extends KalturaQuizUserEntryBaseFilter
 {
+	/**
+	 * @var KalturaNullableBoolean
+	 */
+	public $isAnonymous;
+
+	public function getListResponse(KalturaFilterPager $pager, KalturaDetachedResponseProfile $responseProfile = null)
+	{
+		if(!is_null($this->isAnonymous))
+		{
+			if(KalturaNullableBoolean::toBoolean($this->isAnonymous)===false)
```

Comment:
Please handle also case that (KalturaNullableBoolean::toBoolean($this->isAnonymous)===**true**).